config/addresses.go

```diff
@@ -21,10 +24,11 @@ type Addresses struct {
 // NewAddresses returns Addresses with values set to their defaults.
 func NewAddresses() Addresses {
 	return Addresses{
-		Admin:   "/ip4/127.0.0.1/tcp/3002",
-		Finder:  "/ip4/0.0.0.0/tcp/3000",
-		Ingest:  "/ip4/0.0.0.0/tcp/3001",
-		P2PAddr: "/ip4/0.0.0.0/tcp/3003",
+		Admin:         "/ip4/127.0.0.1/tcp/3002",
+		Finder:        "/ip4/0.0.0.0/tcp/3000",
+		FinderWebpage: "https://web.cid.contact/",
```

Comment:
We should parameterise this and set it per environment. What are the impacts of hard coding this for:
* people whom want to run their own indexer under some other domain
* `dev` vs `prod` deployment

Reply:
this is meant as a default - are these structs not loaded from per-instance configs?

Reply:
If the default is not overridden in corresponding configs e.g. [here](https://github.com/filecoin-project/storetheindex/blob/main/deploy/manifests/dev/us-east-2/tenant/storetheindex/indexer-config.yaml#L14) both dev and prod end using the same path. Is that the intention? If so, how would the new UI know which API to call?

Reply:
https://github.com/willscott/cid.contact/blob/main/index.html#L247-L248

Reply:
I'm happy to also change the dev config you point to to have it embed a different root

Reply:
I see thank you

Reply:
I honestly think we should rethink this and pull UI stuff out. but for now iframe seems like a reasonable approach to get  the new UI rolled out.